parts/linux/cloud-init/artifacts/init-aks-custom-cloud.sh

```diff
@@ -31,12 +31,10 @@ cloud-init status --wait
 repoDepotEndpoint="{{AKSCustomCloudRepoDepotEndpoint}}"
 sudo sed -i "s,http://.[^ ]*,$repoDepotEndpoint,g" /etc/apt/sources.list
 
-# Disable systemd-timesyncd and install chrony and uses local time source
+# Disable systemd-timesyncd and config chrony uses local time source
 systemctl stop systemd-timesyncd
 systemctl disable systemd-timesyncd
 
```

Comment:
@xizhamsft for 1804 the VHD always have NTP enabled, should well disable it? Did you observe any issue for 1804 in lx/rx?

Reply:
reverted changes to this file. Xiaofang will handle it.